test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -1183,10 +1184,11 @@ public void testLogEntrySyncWithFSMChangeAndWithAckDrop() throws Exception {
         expectedAckTimestamp.set(srcDataRuntime.getAddressSpaceView().getLogTail());
 
         // Block until the expected ACK Timestamp is reached
+        log.info("Here 2");
```

Comment:
nit: is this for debugging? just noticed it when rebasing the change

Reply:
yes